File: app/src/main/java/com/example/chatgptapp/sql/MyDatabaseHelper.java
package com.example.chatgptapp.sql;

import android.database.sqlite.SQLiteDatabase;
import android.database.sqlite.SQLiteOpenHelper;
import android.util.Log;

import com.example.chatgptapp.MainActivity;

public class MyDatabaseHelper extends SQLiteOpenHelper {
    private static final String CREATE_TABLE = "CREATE TABLE chatList ("
            + "id INTEGER PRIMARY KEY AUTOINCREMENT, "
            + "type INTEGER, "
            + "content TEXT)";

    public MyDatabaseHelper( String name, int version) {
        super(MainActivity.Instance.getApplicationContext(), name, null, version);
    }

    @Override
    public void onCreate(SQLiteDatabase db) {
        db.execSQL(CREATE_TABLE);
        Log.i("MyDatabaseHelper", "Database created successfully");
    }

    @Override
    public void onUpgrade(SQLiteDatabase db, int oldVersion, int newVersion) {
        // Implement upgrade logic here if needed
    }
}

File: app/src/main/java/com/example/chatgptapp/sql/MyDatabaseHelper.kt
package com.example.chatgptapp.sql

import android.content.Context
import android.database.sqlite.SQLiteDatabase
import android.database.sqlite.SQLiteOpenHelper
import android.util.Log
import android.widget.Toast

class MyDatabaseHelper(val context: Context, name: String, version: Int) :
    SQLiteOpenHelper(context, name, null, version) {
    private val createTable = "create table chatList (" +
            " id integer primary key autoincrement," +
            "type integer," +
            "content text)"

    override fun onCreate(db: SQLiteDatabase) {
        db.execSQL(createTable)
        Log.i("MyDatabaseHelper","database create success")
    }

    override fun onUpgrade(db: SQLiteDatabase, oldVersion: Int, newVersion: Int) {
    }
}